### ISSUE TITLE
ReferenceError: __slice is not defined

### ISSUE THREAD

**Issue by jackwilsdon** [LukePammant/pretty-logger#3]
I am getting the following error when trying to use `pretty-logger`:

```
node_modules\pretty-logger\pretty-logger.js:33
         msg = arguments[0], args = 2 <= arguments.length ? __slice.call(argum
                                                                    ^
ReferenceError: __slice is not defined
    at Logger.PrettyLogger.BasicLogger.info (node_modules\pretty-logger\pretty-logger.js:33:76)
    at Object.<anonymous> (main.js:14:5)
    at Module._compile (module.js:460:26)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Function.Module.runMain (module.js:501:10)
    at startup (node.js:129:16)
    at node.js:814:3
```

And here is the code I'm using:

``` js
var Logger = require('pretty-logger');

var loggerConfig = {
    showTimestamp: true,
    prefix: "[" + process.pid.toString() + "]"
};

var log = new Logger(loggerConfig);

log.info("Hello, %s!", "World");
```

**Comment by LukePammant** [comment]
I have found a solution for this and will push it out soon.

**Comment by LukePammant** [comment]
@jackwilsdon, thank you for your detailed issue report. 

I have fixed the issue and pushed it up to GitHub and released a version 1.2 on NPM. Please feel free to download and test.
